.gitignore

```diff
@@ -1,2 +1,7 @@
 target/
 
+.idea/
+
+.*sw*
```

Comment:
It's better to add these rules to your global .gitignore.

Reply:
you mentioned on the training as well.